src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs

```diff
@@ -161,6 +180,49 @@ public void run(ChocolateyConfiguration configuration)
                                     xw.WriteAttributeString("version", packageResult.Package.Version.ToString());
                                 }
 
+                                if (configuration.ExportCommand.IncludeRememberedPackageArguments)
+                                {
+                                    var pkgInfo = _packageInfoService.get_package_information(packageResult.Package);
+                                    configuration.Features.UseRememberedArgumentsForUpgrades = true;
+                                    _nugetService.set_package_config_for_upgrade(configuration, pkgInfo);
+
+                                    //Mirrors the arguments captured in ChocolateyPackageService.capture_arguments()
+
+                                    if (configuration.Prerelease) xw.WriteAttributeString("prerelease", "true");

```

Comment:
These strings, for example `prerelease` are starting to feel a little bit like "magic" strings, as they are being used in multiple places as bare strings. Being used here, as well as when naming the attributes in the `PackagesConfigFilePackageSetting` class.  Should we break these out into a new class, of const string's, or similar, so we can define them once, and use in multiple places?

Reply:
Hmm, why not use the `PackagesConfigFilePackageSetting` (or `PackagesConfigFileSetting`) class to do the xml serialization, that way the strings would be in only one place?

Reply:
Hmm, that is not a bad idea. That would mean that the serialized XML could be used elsewhere. Although, I don't immediately see a need for it anywhere else, but putting it in one of these classes would mean that we "simplify" the command class, and do the work elsewhere.

Reply:
I've switched to serializing that class to create the xml. It required adding members so the boolean options are not serialized if they are not specified, but otherwise I like the implementation.